guardian/src/main/java/org/rfcx/guardian/guardian/api/CheckGuardianExistedApi.kt

```diff
@@ -0,0 +1,44 @@
+package org.rfcx.guardian.guardian.api
+
+import android.content.Context
+import android.os.Handler
+import android.os.StrictMode
+import org.rfcx.guardian.guardian.RfcxGuardian
+import org.rfcx.guardian.guardian.manager.getTokenID
+import org.rfcx.guardian.utility.http.HttpGet
+
+object CheckGuardianExistedApi {
+
+    private lateinit var httpGet: HttpGet
+
+    fun isExisted(context: Context, guid: String, callback: CheckGuardianCallback) {
```

Comment:
Naming: slightly better wording would be `GuardianCheckApi`, and the function can be `exists()`